### ISSUE TITLE
stash >> pip install Shopify // TypeError("a bytes-like object is required, not 'str'")

### ISSUE THREAD

**Issue by kadabra2** [Shopify/shopify_python_api#322]
When I try to run 'pip install ShopifyAPI' on Pythonista using stash, I get the error below:
![IMG_F28DF40794FE-1](https://user-images.githubusercontent.com/14969583/58902680-1a6c7080-86f3-11e9-8519-2b8d0bc4a431.jpeg)

**Comment by katiedavis** [comment]
@kadabra2 I'm not familiar with Pythonista, but is it possible it's running an incompatible version of Python?

**Comment by kadabra2** [comment]
Yes, When I run this command using Python 2.7 it works fine. However, my code use some other API that require Python3. That's why I try to figure out how to resolve the issue mentioned above.
